toppra/algorithm/reachabilitybased/reachability_algorithm.py

```diff
@@ -352,11 +352,12 @@ def compute_parameterization(self, sd_start, sd_end, return_data=False):
                 x_next = xs[i] + 2 * deltas[i] * us[i]
                 x_next = max(x_next - TINY, 0.9999 * x_next)
                 xs[i + 1] = min(K[i + 1, 1], max(K[i + 1, 0], x_next))
-                logger.debug(
-                    "[Forward pass] u[{:d}] = {:f}, x[{:d}] = {:f}".format(
-                        i, us[i], i + 1, xs[i + 1]
+                if logger.isEnabledFor(logging.DEBUG):
+                    logger.debug(
```

Comment:
I think `logger.debug("[Forward pass] u[%d] = %f, x[%d] = %f", i,us[i], i+1, xs[i+1])` will have the same effect.

Reply:
This will do anyway.

Thanks for the PR!